Portal/View/UIKit/PullToRefreshable.swift

```diff
@@ -32,20 +32,22 @@ extension PullToRefreshable where Self : UIScrollView {
 
 extension PullToRefreshable where Self : UIView {
     
-    func configure(pullToRefresh properties: RefreshProperties<ActionType>) {
+    func configure(pullToRefresh properties: RefreshProperties<ActionType>, offset: CGPoint) {
+        scrollView.refreshControl?.endRefreshing()
         let refreshControl = UIRefreshControl()
         refreshControl.attributedTitle = properties.title
         scrollView.refreshControl = refreshControl
         
         switch properties.state {
             
         case .searching:
-            scrollView.contentOffset = CGPoint(x:0, y: -refreshControl.frame.size.height)
+            scrollView.contentOffset = CGPoint(x: 0, y: -refreshControl.frame.size.height)
             refreshControl.beginRefreshing()
             
         case .idle(let message):
-            _ = refreshControl.on(event: .valueChanged, dispatch: message)
```

Comment:
Why this didn't work?

Reply:
Oh I see, I didn't work because we were not forwarding the refreshControl mailbox to the `PullToRefreshable` mailbox.

The following code should work

```swift
case .idle(let message):
    let refreshControlMailbox = refreshControl.on(event: .valueChanged, dispatch: message)
    refreshControlMailbox.forward(to: mailbox)
```